### PR TITLE
add quote reply shortcut

### LINKED ISSUE #1396: Support quote reply
~1. support `@`  autocomplete~
2. support `Quote reply` shortcut

### PR DESCRIPTION
**Pull request by shaodahong**
Closed #1396 

@RMacfarlane Hi, this my first pull request, and `quote reply` need a new icon, so what I to do?

### CODE REVIEW

**Comment by shaodahong** [comment]
now I use `commentIcon`, but I don't know this icon suitable for it

File: preview-src/comment.tsx [hunk 1]
```diff
@@ -64,6 +65,7 @@ export function CommentView(comment: Props) {
 		onMouseLeave={() => setShowActionBar(false)}
 	>{ ((canEdit || canDelete) && showActionBar)
```
Comment (RMacfarlane) [inline preview-src/comment.tsx]: Let's update this to just `showActionBar` so you can quote reply even if you don't have permissions to edit or delete the comment

**Comment by shaodahong** [comment]
and have a question, I don't  find `.editorconfig` or '.prettierrc', so document changed when I format document, but I don't modify the document

**Comment by RMacfarlane** [comment]
Yeah, there is no `.editorconfig` or `.prettierrc` for this project. There are also no linting rules for this formatting. Prettier and the built in VSCode JSON formatter both format arrays in the same way, so I think the change is OK. We had just never run format document on this before, and have the `Format on Save` setting turned off.

**Pull request merged**